src/applications/gi-sandbox/containers/SearchResultCard.jsx

```diff
@@ -110,7 +110,8 @@ export function SearchResultCard({
         </p>
         <div className="vads-u-margin-bottom--2">
           <RatingsStars rating={ratingAverage} />
-          {Math.round(10 * ratingAverage) / 10} of 5)
+          {Math.round(10 * ratingAverage) / 10} of 5) by {ratingCount} Veteran
```

Comment:
Shouldn't there be an opening paren?  ie. (3 of 5)

Reply:
Fixed